docs/rules/object-curly-newline.md

```diff
@@ -19,6 +19,7 @@ Or an object option:
 
 * `"multiline": true` (default) requires line breaks if there are line breaks inside properties or between properties
 * `"minProperties"` requires line breaks if the number of properties is more than the given integer
+* `"consistent"` requires that either both curly braces, or neither, directly enclose newlines
```

Comment:
I'm not sure. I suppose if we can't come up with a better solution, it would be fine to keep this solution as long as we clearly document it.